web/packages/shared/components/FieldInput/FieldInput.tsx

```diff
@@ -57,6 +57,7 @@ const FieldInput = forwardRef<HTMLInputElement, FieldInputProps>(
       spellCheck,
       readonly = false,
       toolTipContent = null,
+      tooltipSticky = false,
```

Comment:
Do you know what's going to be the fate of sticky tooltips? [I asked Kenny about them](https://gravitational.slack.com/archives/C07KG2RFA/p1736262739708629) and [he wanted to remove them](https://docs.google.com/document/d/10gbnTKPJ-JJ_o4jNMrWaNbHF66sC3PBsr5XZbgk-U8g/edit?tab=t.0#heading=h.lk0dliulpirk).

Reply:
...and I have just spoken to @rishibarbhaya-design  and he told me that if the tooltip contains links, it can be sticky.

Reply:
OK, so there's a bigger change to the tooltip behavior that will be upcoming (https://gravitational.slack.com/archives/C07KG2RFA/p1737734143523889), but I'm going to keep this change as it is for now, as it unblocks the follow-up PR. The tooltip behavior itself will need to be addressed separately.